client/packages/system/src/Vaccination/Components/VaccineCardTable.tsx

```diff
@@ -73,55 +73,42 @@ const useStyleRowsByStatus = (
       )
       .map(row => row.id);
 
-    setRowStyles(
+    updateRowStyles(
       doneRows,
       {
         '& td:not(:first-child)': {
           backgroundColor: `${theme.palette.background.success} !important`,
         },
-      },
+      }
       // Parameter to prevent the previous setRowStyles from being
       // reset/overwritten
-      false
     );
-    setRowStyles(
-      nonClickableRows,
-      {
-        '& td': {
-          cursor: 'default',
-        },
-        backgroundColor: 'white !important',
+    updateRowStyles(nonClickableRows, {
+      '& td': {
+        cursor: 'default',
       },
-      false
-    );
-    setRowStyles(
-      allRows,
-      {
-        backgroundColor: 'white !important',
-        boxShadow: 'none',
-        '& td': {
-          borderBottom: `${BORDER_STYLE} !important`,
-        },
-        '& td:nth-child(2)': {
-          borderLeft: BORDER_STYLE,
-        },
-        '& td:first-child': {
-          borderBottom: 'none !important',
-          fontWeight: 'bold',
-        },
+      backgroundColor: 'white !important',
+    });
+    updateRowStyles(allRows, {
+      backgroundColor: 'white !important',
+      boxShadow: 'none',
+      '& td': {
+        borderBottom: `${BORDER_STYLE} !important`,
       },
-      false
-    );
-    setRowStyles(
-      lastOfEachAgeRange,
-      {
-        '& td:first-child': {
-          borderBottom: BORDER_STYLE,
-          fontWeight: 'bold',
-        },
+      '& td:nth-child(2)': {
```

Comment:
Oo one more nit... I know we aren't server side rendering, so should be fine, but the could we change this to clear the console warnings?
![Screenshot 2024-10-01 at 9 28 16 AM](https://github.com/user-attachments/assets/9df70191-6925-4370-bb92-6b9be1ac8d27)

Reply:
That's an annoying warning, but fortunately easy to replace :) a89072883a